docs/pages/distribution/publishing-websites.mdx

```diff
@@ -3,12 +3,56 @@ title: Publish websites
 description: Different ways to publish the Expo web app with third-party services.
 ---
 
-import { Terminal } from '~/ui/components/Snippet';
+import { APIBox } from '~/components/plugins/APIBox';
 import { Step } from '~/ui/components/Step';
 import { Tab, Tabs, TabsGroup } from '~/ui/components/Tabs';
+import { Table, TableHead, Row, HeaderCell } from '~/ui/components/Table';
+import { Terminal } from '~/ui/components/Snippet';
+import { YesIcon, NoIcon, PendingIcon } from '~/ui/components/DocIcons';
 
 A web app created using Expo can be served locally for testing out the production behavior. Once the testing phase checks out, you can choose from a variety of third-party services to host it.
 
+## Output targets
+
+Expo supports three output targets for web apps. The output target can be configured in the **app.json** file:
+
+```json app.json
+{
+  "expo": {
+    "web": {
+      "output": "server",
+      "bundler": "metro"
+    }
+  }
+}
+```
+
+> If you are using Expo Web without Expo Router, only `single` is supported
+
+| Output             | Expo Router | API Routes  |  Description                                                                                                                   |
```

Comment:
I would like advise on this table. I'm trying to explain the outputs, while also showing the API routes only works with 1 output, while also explaining that these are Expo Router only, and Metro/Webpack don't have these features.

Its got a lot going on

Reply:
I think the table looks fine. To add more context of what you are trying to explain, let's add it above this table.